bodhi/server/consumers/masher.py

```diff
@@ -636,7 +636,8 @@ def update_comps(self):
             util.cmd(['git', 'clone', comps_url, comps_dir], os.path.dirname(comps_dir))
 
         util.cmd(['git', 'pull'], comps_dir)
-        util.cmd(['make'], comps_dir)
+        target = 'target-%s.xml' % self.release.branch
```

Comment:
It seems like some unit tests are setting ```self.release``` to a ```unicode``` object. I presume that in production ```self.release``` is a ```bodhi.server.models.Release``` instance, but I'm not 100% sure of that. In any case, this line seems to cause a handful of unit tests to fail.

Reply:
Yeah, I goofed it.  Just rebased.